Simulator/player.py

```diff
@@ -1868,6 +1894,22 @@ def won_round(self, damage):
                     fortune_returns[self.fortune_loss_streak])
                 self.fortune_loss_streak = 0
 
+    def add_augment(self, augment:Augment)-> bool:
+        """ Adds an augment to the player's list of augments
+        Parameters:
+            augment (Augment): The augment to be added
+        Returns:
+            bool: True if the augment was successfully added, False otherwise
+        """
+        if len(self.augments) >= 3:
+            return False
+        self.augments.append(SelectedAugmentConfig(augment, self.round))
```

Comment:
If all we are doing in this method is appending an augment. This should probably be done in the game round when we do a augment selection round.